### PR TITLE
feat: Add "What's new" page

### PR DESCRIPTION
**Pull request by Danktuary**
**Please describe the changes this PR makes and why it should be merged:**
This adds a "What's new" page for the release of discord.js v13. It lists a few site changes, the new pages, and the notable updated content.

### CODE REVIEW

**Comment by J-Human** [comment]
It would be nice if there are automated bots/workflows that can detect new pages and generate new content + PR it but I think it's too complicated. But that aside, I love it!

File: guide/whats-new.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+- [Buttons](/interactions/buttons.md): Building, sending, and receiving buttons
+- [Select menus](/interactions/select-menus.md): Building, sending, and receiving select menus
+- [Threads](/popular-topics/threads.md): Creating and managing threads
+
+### Updated
+
+- [Voice](/voice/): Rewritten to use the [`@discordjs/voice`](https://github.com/discordjs/voice) package
+- Commando: Removed, to be replaced with a guide on [Sapphire](https://github.com/discordjs/guide/pull/711)
+- [Command handling](/command-handling.md): Updated to use slash commands
+	- Obselete sections have been removed
```
Comment (kyranet) [inline guide/whats-new.md:59]:
```suggestion
	- Obsolete sections have been removed
```

**Review by kyranet — CHANGES_REQUESTED** [review]
Just one thing, LGTM otherwise!

**Comment by WiseDevHelper** [comment]
A few questions:
- Going forward, will content be added to by users in their PRs?
- Will this page adopt a changelog style?
- Will changes too old be removed?
- Could this possibly be re-implemented or replaced with components later?

**Comment by WiseDevHelper** [comment]
This resolves #664

**Review by kyranet — APPROVED** [review]

**Comment by Danktuary** [comment]
> A few questions:
> 
> * Going forward, will content be added to by users in their PRs?
> * Will this page adopt a changelog style?
> * Will changes too old be removed?
> * Could this possibly be re-implemented or replaced with components later?

1. Probably not, this page is to raise awareness about v13's release and the new content we have relevant to it. Consider it temporary
2. It'll stay as is
3. Anything here will be staying until the page is removed
4. This may eventually be reimplemented as a changelog style page with components.

> This resolves #664

It doesn't; I'll make a better implementation to resolve that issue.

**Comment by Brennvo** [comment]
I'm a bit confused. I just started using Discord.js a couple of days ago, so I'm just finding this guide. I refreshed the guide and see the docs mentioning v13, but I can't seem to install v13. I checked the Discord.js repo itself, and it seems that it's still on v12. When is v13 expected to be released?

**Comment by almostSouji** [comment]
We are still working on the finishing touches for v13, but want the guide to be up to speed before a new major version. The v12 guide is archived and available at <http://v12.discordjs.guide/>. If you want to check out v13 before its official release you can install `npm i discord.js@dev`.

If you require more in-detail help check out the specific feature support channels on our discord server at https://discord.gg/djs

**Pull request merged**